### PR TITLE
Remove InAgencyPermissions column from AgentAgencyRelation

### PR DESCRIPTION
**Pull request by konstyefremov**
Resolves https://github.com/happy-travel/agent-app-project/issues/366
Resolves https://github.com/happy-travel/agent-app-project/issues/413

### CODE REVIEW

File: Api/Services/Agents/AgentRegistrationService.cs [hunk 1]
```diff
@@ -140,7 +140,6 @@ private Task AddAgentAgencyRelation(Agent agent, AgentAgencyRelationTypes relati
             {
                 AgentId = agent.Id,
                 Type = relationType,
-                InAgencyPermissions = permissions,
```
Comment (mingazhev) [inline Api/Services/Agents/AgentRegistrationService.cs:143]: Should not you delete a method parameter `permissions` then?

Comment (konstyefremov) [reply Api/Services/Agents/AgentRegistrationService.cs:143]: Removed this and checked again the code for remnants of old code.

**Comment by konstyefremov** [comment]
Also deleted Permissions Sets file.

**Comment by konstyefremov** [comment]
Also removed Skip from two unit tests on Permission Checks.

**Review by mingazhev — APPROVED** [review]
To merge after code freeze only (!)

**Pull request merged**